### PR TITLE
added --sarif-file

### PR DESCRIPTION
**Pull request by dkourkouzelis**

### CODE REVIEW

**Comment by dkourkouzelis** [comment]
@hghmn I installed tsc and ran "yaml run all" locally. The tests succeed but it also removed the code that I added to index.js. What am I missing? 

```
+++ b/dist/index.js
@@ -20346,7 +20346,6 @@ function run() {
             const SCAN_SUMMARY_OUTPUT_TYPE = core.getInput("scan_summary_output_type");
             const CI_RUN = core.getBooleanInput("ci_run");
             const CI_RUN_TAGS = core.getInput("ci_run_tags");
-            const SARIF_OUTPUT_FILE = core.getInput("sarif_output_file");
             const ADDITIONAL_ARGS = core.getInput("additional_args");
             const EXPORT_SCAN_RESULT_ARTIFACT = core.getBooleanInput("export_scan_result_artifact");
             const ADDITION_OPTIONS = ADDITIONAL_ARGS.split(" ");
@@ -20390,9 +20389,6 @@ function run() {
             if (CI_RUN_TAGS) {
                 options.push(`--ci-run-tags=${CI_RUN_TAGS}`);
             }
-            if (SARIF_OUTPUT_FILE) {
-                options.push(`--sarif-output-file=${SARIF_OUTPUT_FILE}`);
-            }
             if (ADDITIONAL_ARGS && ADDITION_OPTIONS.length > 0) {
                 options.push(...ADDITION_OPTIONS);
             }
```

**Comment by hghmn** [comment]
@dkourkouzelis the changes here were made against the build output instead of the source files.

If you make the related changes from `dist/index.js` to `src/scan.ts`, and re-run `yarn run all`, you should see the changes from source reflected in the build output as well.

**Comment by dkourkouzelis** [comment]
> @dkourkouzelis the changes here were made against the build output instead of the source files.
> 
> If you make the related changes from `dist/index.js` to `src/scan.ts`, and re-run `yarn run all`, you should see the changes from source reflected in the build output as well.

Thank you @hghmn!

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "@actions/http-client": "^2.0.1",
     "@actions/io": "^1.1.2",
     "@actions/tool-cache": "^2.0.1",
-    "@octokit/webhooks-definitions": "^3.67.3"
+    "@octokit/webhooks-definitions": "^3.67.3",
+    "tsc": "^2.0.4"
```
Comment (hghmn) [inline package.json]: [`tsc`](https://www.npmjs.com/package/tsc) is not an official Microsoft package, should be removed from the list of dependencies here. The `tsc` binary is already provided by the `typescript` dev dependencies.

**Review by hghmn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dkourkouzelis) [reply package.json]: I'd be more than happy to if you could guide me on how to do it. I had to install "tsc" to repeat the workflow steps. How can I remove it?

Comment (hghmn) [reply package.json]: The `tsc` addition can be reverted with:

```sh
yarn remove tsc
yarn run all
rm .licenses/npm/tsc.dep.yml
```

**Review by hghmn — APPROVED** [review]

**Pull request merged**